docs/examples/deploying_with_git _deploy.md

```diff
@@ -110,21 +111,17 @@ blocks:
       - name: Push code
         commands:
           - checkout
+          # Using `ssh-keyscan` you specify that your-server.com is a trusted domain
+          # and bypass an interactive confirmation step that would block the job.
           - ssh-keyscan -H your-server.com >> ~/.ssh/known_hosts
           - chmod 600 ~/.ssh/id_rsa_git_deploy
+          # Manually adding the private SSH key to local SSH agent.
```

Comment:
Comments should focus on "why we are doing this" instead of "what we are doing".

Proposal for new copy:

```
Adding the private SSH key to the local SSH agent so it will be available while executing Git push.
```